code/server/public/js/signup.js

```diff
@@ -0,0 +1,34 @@
+function validatePasswords(pw1, pw2) {
+	console.log('validating password');
```

Comment:
The next following console logs will display the user's password on the browser console. We should delete this.